.github/workflows/e2e.yaml

```diff
@@ -158,7 +158,7 @@ jobs:
         if: inputs.suite != 'Nonbehavioral'
         run: |
           AZURE_CLUSTER_NAME=${{ env.CLUSTER_NAME }} AZURE_RESOURCE_GROUP=${{ env.RG_NAME }} make az-creds
-          CLUSTER_NAME=${{ env.CLUSTER_NAME }} AZURE_ACR_NAME=${{ env.ACR_NAME}} TEST_SUITE="${{ inputs.suite }}" GIT_REF="$(git rev-parse HEAD)" make e2etests
+          AZURE_LOCATION=${{ inputs.location }} AZURE_SUBSCRIPTION_ID=${{ secrets.E2E_SUBSCRIPTION_ID }} AZURE_RESOURCE_GROUP=${{ env.RG_NAME }} AZURE_CLUSTER_NAME=${{ env.CLUSTER_NAME }} AZURE_ACR_NAME=${{ env.ACR_NAME}} TEST_SUITE="${{ inputs.suite }}" GIT_REF="$(git rev-parse HEAD)" make e2etests
```

Comment:
This is becoming unreadable. Can you use `env:` block instead?

Reply:
Switched to that style of syntax instead.